package.json

```
@@ -44,7 +45,8 @@
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
     "test-api": "node test.js",
     "test-coverage": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 --reporter lcov node test.js",
```

```suggestion
    "test-coverage": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 --reporter lcov node test.js && type-coverage",
```

Here maybe? Or in `format` or `build`?

---

updated

> format

maybe? it doesn't modify/fix content in the way the other formatters do

> build

Sure, added a post build step